### PR TITLE
Autofocus on `connect` button at start

### PR DESCRIPTION
**Pull request by sunglim**
Until now `connect` button doesn't have focus by default at start. This sucks because we always have to use 'mouse' to click it. After this patch we can just press `Enter` to connect(if we don't change options).

- Set autofocus on 'connect' button at start.
- Create draw_ui.js and move some UI related codes.

TEST=Load app. See all option select is working well.

@RandomlyKnighted  When you can find time.

### CODE REVIEW

**Comment by RandomlyKnighted** [comment]
I should have some time this weekend to test it out.

**Comment by sunglim** [comment]
@RandomlyKnighted  You actually really don't need to run it by yourself.( I think it is burden).  On commit description, I updated `TEST=` that I tested. If you think there should be more test. Feel free to ask me. In other word, Leaving just comments about code(for example, clean code. better design) is enough.

File: app/js/index.js [hunk 1]
```diff
@@ -149,79 +147,8 @@ window.onload = function() {
   t.decorate(document.querySelector('#terminal'));
 
   t.onTerminalReady = function() {
+    UI_INSTANCE.OnHtermReady();
```
Comment (sunglim) [inline app/js/index.js]: @RandomlyKnighted  I'm really sorry that I still can't find your comment on this patch.
Could you please leave comment again?

Comment (sunglim) [inline app/js/index.js]: test

File: app/js/index.js [hunk 2]
```diff
@@ -5,8 +5,10 @@
 var inputOutput;
 var self;
 
+var UI_INSTANCE = new DrawUi();
+
 document.addEventListener('DOMContentLoaded', function() {
-  $('#settingsModal').modal('show');
+  UI_INSTANCE.ShowSettingsDialog();
 }, false);
```
[the remark this answers is not in the document]
Comment (sunglim) [reply app/js/index.js:13]: Hi @RandomlyKnighted. Thank you for the comment.

Yes. It makes new dependency. But I want to move UI related code to separate file to easily maintain.
Wdyt?

**Review by sunglim — COMMENTED** [review]
1

File: app/js/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (codewithtyler) [reply app/js/index.js:13]: @sunglim Ah yes I see that now. When I first saw your PR I just had time to skim through the code. Now that I've looked at it in more detail I can see that the new dependency is actually the JS code for pulling interacting with the UI and with hterm. That's fine with me.

**Pull request merged**